jenkins/artifacts/jenkinsfile

```diff
@@ -216,12 +217,16 @@ def updateStatus(def commitId, def statusMsg, def buildUrl, def description, def
     post.setRequestProperty("Authorization", "token ${gitToken}")
     post.getOutputStream().write(message.getBytes("UTF-8"));
     println(new String(post.getOutputStream().toByteArray(), "UTF-8"));
+    println("Job Name --> ${jobName}")
```

Comment:
would it make more sense to move this to the top of the method and exit early before making the post call to github?

Reply:
thats right. fixed.